File: SearchAndFindBackend/SearchAndFind.DTO/Response.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Net;
using System.Text;
using System.Threading.Tasks;

namespace SearchAndFind.DTO
{
    public class Response
    {
        public bool Success { get; set; }
        public string Message { get; set; }
        public bool AuthenticationError { get; set; }
        
        public Response() {
            Success = true;
        }

        public Response( string message)
        {
            Message = message;
            Success = false;
        }
    }
}

File: SearchAndFind-Frontend/app/src/main/java/uy/com/searchandfind/Response.java
package uy.com.searchandfind;

public class Response {
    private boolean Success;
    private String Message;
    private boolean AuthenticationError;

    public Response(){
        Success=true;
    }
    public Response(String message){
        Success=false;
        this.Message=message;
    }

    public boolean isSuccess() {
        return Success;
    }

    public void setSuccess(boolean success) {
        Success = success;
    }

    public String getMessage() {
        return Message;
    }

    public void setMessage(String message) {
        Message = message;
    }

    public boolean isAuthenticationError() {
        return AuthenticationError;
    }

    public void setAuthenticationError(boolean authenticationError) {
        AuthenticationError = authenticationError;
    }
}
